packages/mdx-remote/src/index.tsx

```diff
@@ -1,7 +1,7 @@
 import type { MDXRemoteSerializeResult } from '@guild-docs/types';
 import * as mdx from '@mdx-js/react';
 import React, { ComponentProps, ReactElement, useEffect, useMemo, useState } from 'react';
-import * as runtime from 'react/jsx-runtime.js';
+import * as runtime from 'react/jsx-runtime';
```

Comment:
without this change I got 
![image](https://user-images.githubusercontent.com/7361780/169871733-1a0b5387-1d2d-4aa3-9081-2e54fefc1a39.png)